src/pages/boundary/api-docs/[[...page]].tsx

```diff
@@ -21,15 +23,21 @@ const targetFile = {
 // The path to read from when running local preview in the context of the boundary repository
 const targetLocalFile = '../../internal/gen/controller.swagger.json'
 
-function ApiDocsView({ layoutProps, apiPageProps }) {
+import { SidebarSidecarLayoutProps } from 'layouts/sidebar-sidecar'
+import type { PageWithLayout } from 'pages/_app'
+
+function ApiDocsLayout(props: SidebarSidecarLayoutProps) {
+	return <SidebarSidecarLayout {...props} />
+}
+
+type ApiDocsViewProps = InferGetStaticPropsType<typeof getStaticProps>
+const ApiDocsView: PageWithLayout<ApiDocsViewProps> = ({ apiPageProps }) => {
```

Comment:
It would be interesting to figure out how to get `layoutProps` typed in `getStaticProps` based on a page's specified layout 💭

Reply:
💭 I haven't done any digging, but there's a `satisfies` TS operator coming soon that _might_ be what we need for this